### PR TITLE
feat(cors): add cors middleware

### PR DESCRIPTION
**Pull request by shimoncohen**

### CODE REVIEW

File: projectTemplates/ts-server/src/serverBuilder.ts [hunk 1]
```diff
@@ -33,6 +34,7 @@ export class ServerBuilder {
   }
 
   private registerMiddleware(): void {
+    this.serverInstance.use(cors());
```
Comment (syncush) [inline projectTemplates/ts-server/src/serverBuilder.ts:34]: Move `body-parser` under `cors` middleware under `registerMiddleware()`, also rename `registerMiddleware()` to `registerMiddlewares()`

File: projectTemplates/js-server/package.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
     "standard-version": "^8.0.2"
   },
   "dependencies": {
+    "cors": "^2.8.5",
     "express": "^4.17.1"
   }
 }
```
Comment (syncush) [inline projectTemplates/js-server/package.json:59]: Add the relevant code in the js-server template.

**Review by syncush — CHANGES_REQUESTED** [review]

**Comment by syncush** [comment]
This is not a chore, it is a feature.

**Review by syncush — CHANGES_REQUESTED** [review]
1. You still havn't added the necessary code for the javascript server template
2. this is a feature not a chore, please edit the commit message.

**Review by asafMasa — APPROVED** [review]

**Review by syncush — APPROVED** [review]

**Pull request merged**